crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -427,7 +427,7 @@ def get_node_and_network(node_name)
     @node = NodeObject.find_node_by_name(node_name) if @node.nil?
     if @node
       # If we're in discovery mode, then we have a temporary DHCP IP address.
-      if not ["discovering", "discovered", "hardware-installing", "hardware-installed"].include? @node.state
+      if !["discovering", "discovered"].include?(@node.state)
```

Comment:
> Hrm, good question. And the answer is: "it depends":
[..]
> Which is why I lean towards keeping this.

Thanks for the details. And yes, I agree with you (now) that it is fine to keep it.

> question: should I put all this in the commit message?

Yes please!